client/src/less/bhima-bootstrap.less

```diff
@@ -8,6 +8,44 @@
 
 @border-radius-base: 2px;
 
+@ima-green:           #77a13f;
+@ima-dark-orange:     #e57b14;
+@ima-dark-blue:       #085484;
+@ima-light-orange:    #fdba64;
+@ima-light-blue:      #6fbdee;
+  
+/** @todo move all structure code into less with configurable variables */
+@expanded-offset:  260px;
+@collapsed-offset: 56px;
+@nav-height:       51px;
+
+/**
+* Notifications
+*/
+growl-notifications { 
+  width : 100%;
+  position : fixed;
+  top : 0px;
+  left : @collapsed-offset;
+  z-index : 1000;
+}
+
+.expanded > growl-notifications { 
+  left : @expanded-offset;
+  width : calc(~'100% - (@{expanded-offset} - @{collapsed-offset})');
+}
+
+growl-notifications growl-notification {
+  background : #fff;
+  border-left : 5px solid #9A9A9A;
+  height : @nav-height;
+  padding : 15px 10px;
+  display : block;
+
+  /** @todo width calculation are a hack - the notification doesn't need to be fixed */
+  width : calc(~'100% - @{collapsed-offset}');
```

Comment:
As I understand it it removes the quotation marks \' - This would compile
down to
```css
width : calc(100% - 55px);
```

When I did this without string interpolation less tried to combine the
values resulting in an incorrect percentage vs. a percent take away a pixel
value.

*Steven Fountain*
Democratic Republic of Congo
*Phone: *+243 825763664 <+243%825763664>
*Skype: *stevenfountain

Facebook <https://www.facebook.com/imaworldhealth> | Twitter
<https://twitter.com/imaworldhealth> | LinkedIn
<https://www.linkedin.com/company/441163?trk=tyah&trkInfo=tarId%3A1423079895689%2Ctas%3AIMA%20World%20Health%2Cidx%3A1-1-1>

Donate Today! <https://imaworldhealth.org/donate/>

On Fri, Apr 29, 2016 at 4:15 PM, Jonathan Niles <notifications@github.com>
wrote:

> In client/src/less/bhima-bootstrap.less
> <https://github.com/IMA-WorldHealth/bhima-2.X/pull/370#discussion_r61583800>
> :
>
> > +}
> > +
> > +.expanded > growl-notifications {
> > +  left : @expanded-offset;
> > +  width : calc(~'100% - (@{expanded-offset} - @{collapsed-offset})');
> > +}
> > +
> > +growl-notifications growl-notification {
> > +  background : #fff;
> > +  border-left : 5px solid #9A9A9A;
> > +  height : @nav-height;
> > +  padding : 15px 10px;
> > +  display : block;
> > +
> > +  /** @todo width calculation are a hack - the notification doesn't need to be fixed */
> > +  width : calc(~'100% - @{collapsed-offset}');
>
> Out of curiosity, what does the tilde (~) do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/IMA-WorldHealth/bhima-2.X/pull/370/files/59f2f93778cbe9d67140a6f8811e122dc70da0be#r61583800>
>